<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignIn</class>
 <widget class="QMainWindow" name="SignIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign In</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                          stop:0 #f0f4f8, stop:1 #dbeafe);
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>135</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="minimumSize">
           <size>
            <width>241</width>
            <height>104</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>341</width>
            <height>114</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
            font: bold 36pt &quot;Segoe UI&quot;;
            color: #1e3a8a;
            padding: 20px;
            background-color: #ffffff;
            border-radius: 15px;
           </string>
          </property>
          <property name="text">
           <string>Sign In</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="usernameLabel">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="username_input">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
          QLineEdit {
           font: 12pt &quot;Segoe UI&quot;;
           background-color: #f0f0f0;
           border: 1px solid #ccc;
           border-radius: 6px;
           padding: 5px 8px;
          }
         </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="passwordLabel">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="passwordLayout">
          <item>
           <widget class="QLineEdit" name="password_input">
            <property name="minimumSize">
             <size>
              <width>360</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
            QLineEdit {
             font: 12pt &quot;Segoe UI&quot;;
             background-color: #f0f0f0;
             border: 1px solid #ccc;
             border-radius: 6px;
             padding: 5px 8px;
            }
           </string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Eye_Button">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
            QPushButton {
             background-color: transparent;
             border: none;
             border-image: url(:/images/images/closed_eye.png);
            }
            QPushButton:checked {
             border-image: url(:/images/images/open_eye.png);
            }
           </string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonsLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Back_Button">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>45</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
              QPushButton {
                  background-color: #1d4ed8;
                  color: white;
                  font: 14pt &quot;Segoe UI&quot;;
                  border-radius: 10px;
                  padding: 10px;
              }
              QPushButton:hover {
                  background-color: #2563eb;
              }
              QPushButton:pressed {
                  background-color: #1e40af;
              }
             </string>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="middleSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="LogIn_Button">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>45</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
              QPushButton {
                  background-color: #1d4ed8;
                  color: white;
                  font: 14pt &quot;Segoe UI&quot;;
                  border-radius: 10px;
                  padding: 10px;
              }
              QPushButton:hover {
                  background-color: #2563eb;
              }
              QPushButton:pressed {
                  background-color: #1e40af;
              }
             </string>
          </property>
          <property name="text">
           <string>Sign In</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="bottomSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>135</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
